### PR TITLE
Sprint 28 TLT-1840 Import files command

### PR DESCRIPTION
**Pull request by douglashall**

### CODE REVIEW

File: file_service/management/commands/import_files.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        failed_count = 0
+        while self.canvas_progress_urls:
+            time.sleep(2)
+            completed_imports = []
+            failed_imports = []
+            for progress_url in self.canvas_progress_urls:
+                progress = json.loads(SDK_CONTEXT.session.request('GET', progress_url).text)
+                workflow_state = progress['workflow_state']
+                if workflow_state == 'completed':
+                    completed_imports.append(progress_url)
```
Comment (sapnamysore) [inline file_service/management/commands/import_files.py:83]: Don't you need to increment the completed_imports count here? Never mind, I thought you have a counter for completed counts as well...

File: file_service/management/commands/import_files.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    def _import_isite(self, keyword, canvas_course_id):
+        export_file_url = self._get_export_s3_url(keyword)
+        logger.info("Importing iSites file export from %s to Canvas course %s", export_file_url, canvas_course_id)
+        response = json.loads(content_migrations.create_content_migration_courses(
+            SDK_CONTEXT,
+            canvas_course_id,
+            'zip_file_importer',
+            settings_file_url=self._get_export_s3_url(keyword),
+            settings_folder_id=self._get_root_folder_for_canvas_course(canvas_course_id)['id']
+        ).text)
```
Comment (sapnamysore) [inline file_service/management/commands/import_files.py:120]: I don't see settings_folder_id used anywhere. Was it intended to be logged?

File: file_service/management/commands/migrate_files.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+from django.core.management.base import BaseCommand, CommandError
+from django.core.management import call_command
+
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'Imports iSites file repository export to Canvas'
```
Comment (sapnamysore) [inline file_service/management/commands/migrate_files.py]: Should this read 'Migrate iSites file ....'  to distinguish it from  the help in import_files?

**Comment by sapnamysore** [comment]
Looks good to me! Maybe devops can take a look too. 
BTW, shouldn't this be merged to develop first instead of master?(this PR is opened against master)

File: file_service/management/commands/import_files.py (hunk 2 above)
Comment (douglashall) [inline file_service/management/commands/import_files.py:120]: `settings_folder_id` is a required parameter for the content migration API endpoint. https://canvas.instructure.com/doc/api/content_migrations.html#method.content_migrations.create

File: file_service/management/commands/migrate_files.py (hunk 3 above)
Comment (douglashall) [inline file_service/management/commands/migrate_files.py]: Yes.

**Pull request merged**